spec/support/ssl_helper.rb

```diff
@@ -26,7 +26,8 @@ def generate_key
       when 2
         generate_ecdsa_key('secp384r1')
       when 3
-        generate_ecdsa_key('secp521r1')
+        # generate_ecdsa_key('secp521r1')
```

Comment:
Is this comment needed?

Reply:
These keys where failing against staging when I started developing. I'll give it another go.

@cpu do you know anything about that?

Reply:
@unixcharles Is this something that was working and broke with your V2 work? AFAIK we've never supported secp521r1: https://github.com/letsencrypt/boulder/blob/fa5c9176655d9fa8dfca188de08bd5373aca422f/goodkey/good_key.go#L183-L195

Reply:
@unixcharles I don't believe secp521r1 has ever been supported by Let's Encrypt.